backend/wmg/pipeline/summary_cubes/calculate_markers.py

```diff
@@ -233,6 +246,9 @@ def _query_target(
         level_values = np.array(list(n_cells_index_context.get_level_values(level)))
         filt_ncells = np.logical_and(filt_ncells, np.in1d(level_values, target_filters[level + "s"]))
 
+    if not np.any(filt) or not np.any(filt_ncells):
```

Comment:
An extra safety check - if for whatever reason the target population is not found in the context, raise an error and move on to the next cell type.